File: Libraries/Common/Models/UserLoginInfoModel.cs
namespace System.Share.Models
{
    public class UserLoginInfoModel
    {
        public int UserId { get; set; }
        public string? UserName { get; set; }
        //public string? FullName { get; set; }
        public string? Provider { get; set; }
        public string? Email { get; set; }
        public List<string>? Roles { get; set; }
        public string? AvatarUrl { get; set; }
    }
}

File: webcore.client/src/app/core/models/user-login-info.model.ts
import { INavData } from "@coreui/angular";
export interface UserLoginInfoModel {
  userId: number;
  userName: string;
  provider: string;
  email: string;
  roles: string[];
  avatarUrl?: string;
  isAdmin: boolean;
  navItems?: INavData[];
}
